app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -592,4 +598,45 @@ object TelemetryWrapper {
                 .extra(Extra.TO, to.toString())
                 .queue()
     }
+
+    @JvmStatic
+    fun setDefaultSearchEngineEvent(source: String) {
+        TelemetryEvent.create(Category.ACTION, Method.SAVE, Object.SEARCH_ENGINE_SETTING)
+                .extra(Extra.SOURCE, source)
+                .queue()
+    }
+
+    @JvmStatic
+    fun openSearchSettingsEvent() {
+        TelemetryEvent.create(Category.ACTION, Method.OPEN, Object.SEARCH_ENGINE_SETTING).queue()
+    }
+
+    @JvmStatic
+    fun menuRemoveEnginesEvent() {
+        TelemetryEvent.create(Category.ACTION, Method.REMOVE, Object.SEARCH_ENGINE_SETTING).queue()
+    }
+
+    @JvmStatic
+    fun menuRestoreEnginesEvent() {
+        TelemetryEvent.create(Category.ACTION, Method.RESTORE, Object.SEARCH_ENGINE_SETTING).queue()
+    }
+
+    @JvmStatic
+    fun menuAddSearchEngineEvent() {
+        TelemetryEvent.create(Category.ACTION, Method.NEW, Object.SEARCH_ENGINE_SETTING).queue()
```

Comment:
"New" is a bit confusing. Wouldn't "Show" be more descriptive? From the code it looks like this happens whenever the screen is shown?